src/TestRunner.bs

```diff
@@ -40,7 +43,7 @@ public function new(args = {})
   ' Internal properties
   m.config.testsDirectory = config.testsDirectory
 
-  m.logger = Logger(m.config)
+  m.logger = SLogger(m.config)
```

Comment:
I was testing the build against a client codebase and `Logger` clashes with their own definition. `Logger` is common nomenclature so we should rename. Possibly worth prefixing all global Rooibos defs to avoid collisions?